packages/kbn-securitysolution-hook-utils/tsconfig.json

```diff
@@ -3,7 +3,8 @@
   "compilerOptions": {
     "declaration": true,
     "declarationMap": true,
-    "outDir": "target",
+    "emitDeclarationOnly": true,
+    "outDir": "target_types",
```

Comment:
Below did you wan this like the others?

```json
  "exclude": [
    "**/__fixtures__/**/*"
  ]
````

Reply:
no fixtures for now 😄